{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -140,3 +140,146 @@ extend-unsafe-fixes = [
 
 [tool.ruff.lint.isort]
 force-single-line = true
+
+[tool.uv]
+dev-dependencies = [
```

Comment:
Might want to use the dependency groups here

Reply:
Mist [PEP-735](https://peps.python.org/pep-0735/) but an easy adjustment